stdlib/os/__init__.pyi

```diff
@@ -1072,3 +1080,4 @@ if sys.version_info >= (3, 12) and sys.platform == "linux":
     CLONE_SYSVSEM: int
     CLONE_THREAD: int
     CLONE_VM: int
+    def unshare(flags: int) -> None: ...
```

Comment:
This was related to https://github.com/python/typeshed/pull/10757 but I thought doesn't required a separate PR.